implementation/contracts/deposit/DepositUtils.sol

```diff
@@ -367,6 +367,17 @@ library DepositUtils {
         _d.lastRequestedDigest = bytes32(0);
     }
 
+
+    /// @notice     Get the starting percentage of the bond at auction.
+    /// @dev        This will return the same value regardless of collateral price.
+    /// @return     The percentage of the InitialCollateralizationPercent that will result
+    ///             in a 100% bond value base auction given perfect collateralization.
+    function getAuctionBasePercentage(Deposit storage _d) internal view returns (uint256) {
+        ITBTCSystem _sys = ITBTCSystem(_d.TBTCSystem);
+        uint256 initialCollateralizedPercent = _sys.getInitialCollateralizedPercent();
```

Comment:
We have to store this at deposit creation, otherwise the existing deposit's behavior changes based on governance actions :/

Reply:
derp.